desktop/cmp/treemap/TreeMapModel.js

```diff
@@ -287,10 +288,18 @@ export class TreeMapModel extends HoistModel {
      * b) 'balanced' attempts to account for outliers by adjusting the normalisation ranges around
      *  the median values. Can result in more defined color differences in a dataset that is skewed
      *  by a few nodes at the extremes.
+     * c) 'wash' ignores the intensity of the heat value, applying a flat positive or negative color.
+     * d) 'none' ignores the heat value altogether, coloring all nodes with the neutral color
      */
     normaliseColorValues(data) {
         const {colorMode, heatField} = this;
-        if (!data.length || colorMode === 'none') return data;
+
+        if (!data.length || colorMode === 'none') {
+            return data.map(it => {
```

Comment:
just do the `forEach` here, and mutate the data and then return it (they way we do at bottom)

This `map` is mutating the underlying data, but then returning a new array for no good reason.